src/common/ConfirmationModal.tsx

```diff
@@ -52,8 +52,12 @@ const ConfirmationModal: React.FC<ConfirmationModalProps> = ({ action, handleCli
       onCancel={() => {
         window.history.back();
       }}
-      className={`popupModal${darkModeStatus ? "-dark" : ""} ${darkModeStatus ? "dark" : "light"}-theme${theme[darkModeStatus ? "dark" : "light"]
-        }`}
+      className={`popupModal${darkModeStatus ? "-dark" : ""} ${darkModeStatus ? "dark" : "light"}-theme${
+        theme[darkModeStatus ? "dark" : "light"]
+      }`}
+      maskStyle={{
+        backgroundColor: darkModeStatus ? "rgba(0, 0, 0, 0.50)" : "rgba(87, 87, 87, 0.4)",
```

Comment:
create a variable in customize.scss and apply it on the class of popupModal and popupModal-dark. This would save redundancy.  and in future we can just change that variable to test and play.

Reply:
The classNames prop in modal component does not apply class names to the mask div, which leads to styles not being applied based on the theme to the mask.
![Screenshot from 2023-11-23 16-57-06](https://github.com/tijlleenders/ZinZen/assets/90821253/e795b1d2-7367-4c88-ae2d-81204106c867)